src/sentry/static/sentry/app/constants/index.tsx

```diff
@@ -218,6 +218,10 @@ export const MAX_AUTOCOMPLETE_RELEASES = 5;
 export const DEFAULT_PER_PAGE = 50;
 export const TEAMS_PER_PAGE = 25;
 
+// Limit query length so paginated response headers don't
+// go over HTTP header size limits (4Kb)
```

Comment:
Not relevant for this PR, but something to remember in the future that this is only because we are using `http 1.1` instead of `http2`, as soon as we switch over we can likely remove these limits as headers are compressed.